crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -890,3 +890,149 @@ xxx....vlv....xxx
      .......
      ...T...
 ENDMAP
+
+NAME:   dreamdust_ecumenical_altar_hidden_monk_grotto
+TAGS:   no_monster_gen no_item_gen no_rotate
+KMONS:  p = plant / fungus
+KMONS:  b = butterfly
+KITEM:  ! = never_decay human skeleton, robe mundane, quarterstaff mundane
```

Comment:
an early qstaff is pretty nice loot, you might want to reduce the weight of this vault

Reply:
Thank you for the feedback and sorry for the late response!
I went ahead and halved the weight.

Reply:
In my opinion it's better to leave this at normal weight and remove the weapon.